### DISCUSSION TITLE
Simple search fields should be configurable with a UI

### DISCUSSION THREAD

**Discussion by grantfitzsimmons** [specify/specify7#5564] [Feature Requests]
**Is your feature request related to a problem? Please describe.**
Currently there is no way to configure what the express search function queries. There should be an option to edit which fields are returned in the express search.

**Describe the solution you'd like**
There should be an interface for users to choose what is searched using the express search function.

**Reported By**
Barcelona

**Comment by maxpatiiuk** [comment]
There is an appresource file "ExpressSearchConfig" for configuring the fields that show up in search results.
There doesn't exist a UI for it in Sp7, but changes in Sp6 should be reflected in Sp7.

**Comment by grantfitzsimmons** [comment]
@maxpatiiuk Users will need an interface to make these changes in the future.

**Comment by maxpatiiuk** [comment]
Ben's suggestion:
Make Sp7 express search rely on the query builder UI for defining and customizing queries.
Removes the need for creating and maintaining a separate UI.
Express Search is already using a query builder on the back-end

**Comment by grantfitzsimmons** [comment]
> Express Search is already using a query builder on the back-end

What about the relationships that are returning results? [The ones that must be configured in 6?](https://discourse.specifysoftware.org/t/simple-search-config/183#configuring-related-searches-3)

**Comment by grantfitzsimmons** [comment]
> I wondered if it is possible to have it so that it can open the collection object, rather than the preparation record. 
> Do you know if this is possible?

**Requested By:** RBGE

**Comment by grantfitzsimmons** [comment]
Soraya at Naturhistorisches Museum Bern is looking to customize the simple search configuration in Specify 7 

https://discourse.specifysoftware.org/t/express-search-config-specify-7/1497

**Comment by specifysoftware** [comment]
This issue has been mentioned on **Specify Community Forum**. There might be relevant details there:

https://discourse.specifysoftware.org/t/express-search-config-specify-7/1497/3

**Comment by grantfitzsimmons** [comment]
This has also been requested by CSIRO!

**Comment by bhumikaguptaa** [comment]
CSIC has also requested it!
